features/features.go

```diff
@@ -522,6 +522,14 @@ var (
 				enableForClusterProfile(SelfManaged, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade, configv1.Default).
 				mustRegister()
 
+	FeatureGateNewOLMCatalogdMetas = newFeatureGate("NewOLMCatalogdAPIV1Metas").
+					reportProblemsToJiraComponent("olm").
+					contactPerson("jordank").
+					productScope(ocpSpecific).
+					enhancementPR("https://github.com/openshift/enhancements/pull/1749").
+					enableForClusterProfile(SelfManaged, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
I think you just want to enable everywhere and not make a distinction between cluster profiles
```suggestion
					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Reply:
That would enable us for hypershift, and the new feature flag can't be enable there because it's "parent" NewOLM cannot be enabled there yet.

Reply:
Ack, in which case you have it correct already!